status/raw/status-app-protocols.md

```diff
@@ -0,0 +1,372 @@
+Status Mobile clients are assumed to operate with more resource restrictions and SHOULD use light mode by default.
+
+For the purposes of the rest of this document,
+clients in full mode will be referred to as "full clients" and
+clients in light mode will be referred to as "light clients".
+
+### Subscribing
+
+The application MUST subscribe to receive the traffic necessary for minimal app operation
+and to enable the user functions specific to that instance of the application.
+
+The specific Waku protocol used for subscription depends on the resource-availability of the client:
+1. Filter client protocol, as specified in [12/WAKU2-FILTER](../../waku/standards/core/12/filter.md), allows subscribing for traffic with content topic granularity and is appropriate for resource-restricted subscriptions.
+2. Relay protocol, as specified in [11/WAKU2-RELAY](../../waku/standards/core/11/relay.md), allows subscribing to traffic only with pubsub topic granularity and therefore is more resource-intensive. Relay subscription also allows the application instance to contribute to the overall routing infrastructure, which adds to its overall higher resource usage but benefits the ecosystem.
+
+Full clients SHOULD prefer relay protocol to subscribe to pubsub topics matching the scopes:
+1. Global control
+2. Global content
+3. Community control, for each community of which the app user is a member
+4. Community content, for each community of which the app user is a member
+
+Light clients SHOULD use filter protocol to subscribe only to the content topics relevant to the user.
+
+**Self-addressed messages**
+
+Status clients (full or light) MUST NOT subscribe to topics for messages with self-addressed scopes.
+See [Self-addressed messages](#self-addressed-messages).
+
+**Large messages**
+
+Status clients (full or light) SHOULD NOT subscribe to topics set aside for large messages.
+See [Large messages](#large-messages).
+
+### Publishing
+
+The application MUST publish user and app generated messages via the Waku transport layer.
+The specific Waku protocol used for publishing depends on the resource-availability of the client:
+1. Lightpush protocol, as specified in [19/WAKU2-LIGHTPUSH](../../waku/standards/core/19/lightpush.md) allows publishing to a pubsub topic via an intermediate "full node" and is more appropriate for resource-restricted publishing.
+2. Relay protocol, as specified in [11/WAKU2-RELAY](../../waku/standards/core/11/relay.md), allows publishing directly into the relay routing network and is therefore more resource-intensive.
+
+Full clients SHOULD use relay protocol to publish to pubsub topics matching the scopes:
+1. Global control
+2. Global content
+3. Community control, for each community of which the app user is a member
+4. Community content, for each community of which the app user is a member
+
+Light clients SHOULD use lightpush protocol to publish control and content messages.
+
+**Self-addressed messages**
```

Comment:
you can use h4 titles too `####`

Reply:
Yeah, I think it will be better. Switched to h4 in [9ce3bc5](https://github.com/vacp2p/rfc-index/pull/105/commits/9ce3bc5700324e9172729808b4af9eb68d9a9a91)